web/src/components/profile/ProfileAddressLockedFields.test.tsx

```diff
@@ -36,57 +46,237 @@ describe("ProfileAddressLockedFields", () => {
     );
   };
 
-  it("renders required profile address locked fields", () => {
-    const address = generateAddress({
-      addressLine1: "1111 Home Alone",
-      addressLine2: "",
-      addressMunicipality: generateMunicipality({ displayName: "Allendale" }),
-      addressState: { shortCode: "NJ", name: "New Jersey" },
-      addressZipCode: "00893",
+  describe("NJ business", () => {
+    it("renders required profile address locked fields", () => {
+      useMockBusiness({
+        formationData: generateFormationData({
+          formationFormData: generateFormationFormData({
+            businessLocationType: "NJ",
+          }),
+        }),
+      });
+      const address = generateAddress({
+        addressLine1: "1111 Home Alone",
+        addressLine2: "",
+        addressMunicipality: generateMunicipality({ displayName: "Allendale" }),
+        addressState: { shortCode: "NJ", name: "New Jersey" },
+        addressZipCode: "00893",
+      });
+      renderComponent({ address });
+
+      expect(screen.getByTestId("locked-profileBusinessAddressTooltip")).toBeInTheDocument();
+      const addressLine1 = screen.getByTestId("locked-profileAddressLine1");
+      expect(within(addressLine1).getByText("1111 Home Alone")).toBeInTheDocument();
+      const addressMuniStateZip = screen.getByTestId("locked-profileAddressMuniStateZip");
+      expect(within(addressMuniStateZip).getByText("Allendale, NJ 00893")).toBeInTheDocument();
+      expect(screen.queryByTestId("locked-profileAddressLine2")).not.toBeInTheDocument();
+    });
+
+    it("renders profile address locked fields with Address Line 2", () => {
```

Comment:
[sand] This test is asserting that all possible information is being rendered. I think I'd like this to be the first test and the base case, and then the existing test `renders required profile address locked fields` be changed to assert that addressLine2 is not rendered when the data isn't present.